OpenDreamRuntime/Resources/DreamResourceManager.cs

```diff
@@ -21,7 +21,7 @@ public void Initialize(string jsonPath)
 
         public void SetDirectory(string directory) {
             RootPath = directory;
-            Directory.SetCurrentDirectory(RootPath);
+            //Directory.SetCurrentDirectory(RootPath);
```

Comment:
This line is necessary. It was added because external DLL calls depended on this to be set in order to find/open files in the correct directory.